export/planloader/planloader.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <substrait/common/Io.h>
+
+extern "C" {
+
+using SerializedPlan = struct {
+  // If set, contains a serialized ::substrait::proto::Plan object.
+  char *buffer;
+  // If buffer is set, this is the size of the buffer.
+  int32_t size;
+  // If null the buffer is valid, otherwise this points to an error message.
+  char *errorMessage;
+};
+
+// Since this is actually C code, stick to C style names for exporting.
+// NOLINTBEGIN(readability-identifier-naming)
+
+// Load a Substrait plan (in any format) from disk.
+// Stores the Substrait plan in planBuffer in serialized form.
+// Returns a SerializedPlan structure containing either the serialized plan or
+// an error message.
+SerializedPlan* load_substrait_plan(const char* filename);
```

Comment:
Let's document the arguments too.  I assume `filename` is expected to be a null-terminated C string?

Reply:
Done.